test/native/helpers.js

```diff
@@ -1,239 +1,6 @@
-		// https://github.com/callstack/react-native-testing-library/blob/a010ffdbca906615279ecc3abee423525e528101/src/flushMicroTasks.js#L15-L23.
-		await act( async () => {} );
-
-		return result;
-	} );
-}
-
-/**
- * Initialize an editor for test assertions.
- *
- * @param {Object}                    props               Properties passed to the editor component.
- * @param {string}                    props.initialHtml   String of block editor HTML to parse and render.
- * @param {Object}                    [options]           Configuration options for the editor.
- * @param {import('react').ReactNode} [options.component] A specific editor component to render.
- * @return {import('@testing-library/react-native').RenderAPI} A Testing Library screen.
- */
-export async function initializeEditor( props, { component } = {} ) {
-	const uniqueId = uuid();
-	const postId = `post-id-${ uniqueId }`;
-	const postType = 'post';
-
-	return waitForStoreResolvers( () => {
-		const editorElement = component
-			? createElement( component, { postType, postId } )
-			: internalInitializeEditor( uniqueId, postType, postId );
-
-		const screen = render(
-			cloneElement( editorElement, {
-				initialTitle: 'test',
-				...props,
-			} )
-		);
-
-		// A layout event must be explicitly dispatched in BlockList component,
-		// otherwise the inner blocks are not rendered.
-		fireEvent( screen.getByTestId( 'block-list-wrapper' ), 'layout', {
-			nativeEvent: {
-				layout: {
-					width: 100,
-				},
-			},
-		} );
-
-		return screen;
-	} );
-}
-
 export * from '@testing-library/react-native';
 
-// Custom implementation of the waitFor utility to prevent the issue: https://github.com/callstack/react-native-testing-library/issues/379.
-export function waitFor(
-	cb,
-	{ timeout, interval } = { timeout: 1000, interval: 50 }
-) {
-	let result;
-	let lastError;
-	const check = ( resolve, reject, time = 0 ) => {
-		try {
-			result = cb();
-		} catch ( error ) {
-			lastError = error;
-		}
-		if ( ! result && time < timeout ) {
-			setTimeout(
-				() => check( resolve, reject, time + interval ),
-				interval
-			);
-			return;
-		}
-		resolve( result );
-	};
-	return new Promise( ( resolve, reject ) =>
-		act(
-			() => new Promise( ( internalResolve ) => check( internalResolve ) )
-		).then( () => {
-			if ( ! result ) {
-				reject(
-					`waitFor timed out after ${ timeout }ms for callback:\n${ cb }\n${ lastError.toString() }`
-				);
-				return;
-			}
-			resolve( result );
-		} )
-	);
-}
+export * from './integration-test-helpers';
 
-// Helper for getting the current HTML output of the editor.
-export function getEditorHtml() {
-	if ( ! triggerHtmlSerialization ) {
-		throw new Error( 'HTML serialization trigger is not defined.' );
-	}
-	triggerHtmlSerialization();
-	return serializedHtml;
-}
+// Override `waitFor` export with custom implementation
+export { waitFor } from './integration-test-helpers';
```

Comment:
Does `export * from './integration-test-helpers';` above not achieve the override? Is the explicit named export here is required to override?

Reply:
I had to explicitly export the `waitFor` function because otherwise, I get the following error:
```
TypeError: Cannot redefine property: waitFor
        at Function.defineProperty (<anonymous>)
```

So, looks like the exported functions are not overwritten by default.